docs/website/docs/plus/features/quality/tests.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Test utils
+description: dlt+ Test utils
+keywords: ["dlt+", "data tests", "test"]
+---
+
+## Introduction
+
+dlt+ provides a `pytest` plugin with many useful fixtures and utilities that make writing tests for dlt+ Projects easy.
+
+## Setup
+
+The pytest plugin includes:
+* A set of predefined fixtures (see plugin.py).
+* Additional pytest.ini options (detailed below).
+* A `pytest_config` setup that ensures the test project runs in the correct context and switches to the `tests` profile.
+
+The plugin introduces two `pytest.ini` options, which are automatically set and usually do not need modifications:
+```toml
+[tool.pytest.ini_options]
+dlt_tests_project_path="..."
+dlt_tests_project_profile="..."
+```
+
+### Suggested `conftest.py`
+
+To enable essential fixtures, include the following imports in the `conftest.py`:
+
+```py
+from dlt_plus_tests.fixtures import (
+    auto_preserve_environ as auto_preserve_environ,
+    drop_pipeline as drop_pipeline,
+    autouse_test_storage as autouse_test_storage,
+    auto_cwd_to_tmp_dir as auto_cwd_to_tmp_dir,
+)
+```
+
+By default, no autouse fixtures are enabled, so explicitly importing them is recommended.
+
+Additional cool 😎 utilities for verifying loads, checking table counts, and inspecting metrics can be imported from `utils.py`.
```

Comment:
no emojis pls lol

Reply:
Should we not ddescribe what these utilities are here?

Reply:
I insist!
Also described all available fixtures and utils